src/helix/points.rs

```diff
@@ -0,0 +1,405 @@
+//! Endpoints regarding points
+//!
+//! # Examples
+//!
+//! ```rust,no_run
+//! # use twitch_api2::helix::{HelixClient, points::get_custom_reward_redemption};
+//! # use twitch_api2::types::CustomRewardRedemptionStatus;
+//! # #[tokio::main]
+//! # async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+//! # let token = twitch_oauth2::AccessToken::new("validtoken".to_string());
+//! # let token = twitch_oauth2::UserToken::from_existing(twitch_oauth2::dummy_http_client, token, None, None).await?;
+//! let client = HelixClient::new();
+//! # let _: &HelixClient<twitch_api2::DummyHttpClient> = &client;
+//! let request = get_custom_reward_redemption::GetCustomRewardRedemptionRequest::builder()
+//!     .broadcaster_id("274637212".to_string())
+//!     .reward_id("92af127c-7326-4483-a52b-b0da0be61c01".to_string())
+//!     .status(CustomRewardRedemptionStatus::Cancelled)
+//!     .build();
+//!
+//!
+//! println!("{:?}", &client.req_get(request, &token).await?.data.get(0));
+//! # Ok(())
+//! # }
+//! ```
+use crate::{helix, types};
+use serde::{Deserialize, Serialize};
```

Comment:
Missing re-exports of the types.

```suggestion
use serde::{Deserialize, Serialize};
use get_custom_reward_redemption::{GetCustomRewardRedemptionRequest, CustomRewardRedemption};
use update_redemption_status::{UpdateRedemptionStatusRequest, UpdateRedemptionStatusBody, UpdateRedemptionStatusInformation};
```

Reply:
Changed it to:

```rust
#[doc(inline)]
pub use get_custom_reward_redemption::{CustomRewardRedemption, GetCustomRewardRedemptionRequest};

#[doc(inline)]
pub use update_redemption_status::{
    UpdateRedemptionStatusBody, UpdateRedemptionStatusInformation, UpdateRedemptionStatusRequest,
};
```

per the other modules and rust fmt